### PR TITLE
Graphql coverage

### PR DESCRIPTION
**Pull request by fedinskiy**
Coverage for https://github.com/quarkusio/quarkus/issues/20293

TODOs:
- [x] Merge commits together

### CODE REVIEW

File: http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                .contentType("application/json")
+                .post();
+        Assertions.assertNotEquals(HttpStatus.SC_METHOD_NOT_ALLOWED, response.statusCode());
+        Assertions.assertNotEquals(HttpStatus.SC_INTERNAL_SERVER_ERROR, response.statusCode());
+        Assertions.assertNotEquals(HttpStatus.SC_NO_CONTENT, response.statusCode());
+        Assertions.assertEquals(HttpStatus.SC_BAD_REQUEST, response.statusCode());
+    }
+
+    @Test
+    public void emptyPost() {
```
Comment (pjgg) [inline http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java:38]: Please add here a tag pointing to Quarkus related issue: QUARKUS-1537

File: http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    public static Response sendQuery(String query) {
+        return given().basePath("graphql")
+                .contentType("application/json")
+                .body(query)
+                .post();
+    }
+
+    public static String createQuery(String query) {
+        return new StringBuilder()
```
Comment (pjgg) [inline http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java:54]: Why directly you didn't use a String?

**Review by pjgg — CHANGES_REQUESTED** [review]

File: http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java (hunk 2 above)
Comment (fedinskiy) [reply http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java:54]: I tried to use String, but it became even longer and even less readable. This part of application is not under heavy load, so it's okay if we lost some speed for the sake of readability.

Comment (pjgg) [reply http/graphql/src/test/java/io/quarkus/ts/http/graphql/GraphQLIT.java:56]: Try something like this:

```
 public static String createQuery(String query) {
        return String.format("{\"query\":\"{%s}\"}", query);
    }
```

**Review by pjgg — CHANGES_REQUESTED** [review]
Have a look this comment in order to proceed
https://github.com/quarkus-qe/quarkus-test-suite/pull/446/files#r781051952

**Comment by fedinskiy** [comment]
> Have a look this comment in order to proceed https://github.com/quarkus-qe/quarkus-test-suite/pull/446/files#r781051952

Discussed in private messages, considered less readable.

**Review by pjgg — APPROVED** [review]
LGTM

**Pull request merged**